scss/_base_button.scss

```diff
@@ -97,6 +100,11 @@
   }
 
   &:active,
+  &:active:hover {
+    background-color: $button-active-background-color;
+    border-color: $button-active-border-color;
```

Comment:
Have you tried adding `transition-duration: 0s;`? It seems to fade to a darker colour pretty slowly

Reply:
Nope, but I can do 👍 thanks